lib/rules/simple-modifiers.js

```diff
@@ -0,0 +1,36 @@
+import Rule from './_base.js';
+import { match } from '../helpers/node-matcher.js';
+
+export default class SimpleModifiers extends Rule {
+  visitor() {
+    return {
+      SubExpression(node) {
+        if (!this._isModifier(node)) {
+          return;
+        }
+
+        const firstModifierParam = node.params[0];
+        if (firstModifierParam) {
```

Comment:
It looks like this line isn't fully covered by the tests according to `yarn test:jest --coverage`. I think you'll need to test a modifier with no parameters.

Reply:
Added one more test for 100% coverage.